fetch.bs

```diff
@@ -8449,6 +8597,60 @@ with a <var>promise</var>, <var>request</var>, <var>responseObject</var>, and an
 </div>
 
 
+<h3 id=fetch-later-method>FetchLater method</h3>
+
+<pre class=idl>
+
+dictionary DeferredRequestInit : RequestInit {
+  DOMHighResTimeStamp backgroundTimeout;
```

Comment:
In addition to which type of event to support for this timeout, in our previous PendingBeacon proposal we have also specifies that the user agent has freedom to bundle pending beacon request together, `the beacon is not guaranteed to be sent at exactly this many milliseconds after hidden, because the browser has freedom to bundle/batch multiple beacons, and the browser might send out earlier than specified value.` ([source here](https://github.com/WICG/pending-beacon/blob/0af026cf18312cc612878ada3ad40350f497a006/README.md#properties)). This was for saving resources, and there might be different max time limit for how long a bfcached document can stay in different browser, e.g. Chrome is 10 minutes, while FF is 30 minutes.

Do you think the above can still be added to `fetchLater()`?

Reply:
I reworded this a bit to say that the UA *should* wait for that amount of milliseconds.

Reply:
Forgive me but I can't find the details/definition of `backgroundTimeout` after last change. Is it still here?

Reply:
Yes, see https://whatpr.org/fetch/1647.html#dom-global-fetch-later (#5).
The definition is the same.